platypus/parse.py

```diff
@@ -263,20 +270,19 @@ def process_results(percentage_ids, alignment_lengths, percentage_ids_other,
             # Comparing bit_scores to create outputs
             if vals['a']['bit_score'] == vals['b']['bit_score']:
                 results[i]['equal'] += 1
-                results[i]['summary'].append('%s\t%s\t%s' % (seq_name,
-                                                             subject_id_a,
-                                                             subject_id_b))
+                results[i]['summary_fh'].write('%s\t%s\t%s\n' % (
+                    seq_name, subject_id_a, subject_id_b))
                 db_seqs_counts_a[subject_id_a] += 1
                 db_seqs_counts_b[subject_id_b] += 1
             elif vals['a']['bit_score'] > vals['b']['bit_score']:
                 if not subject_id_b:
                     results[i]['perfect_interest'] += 1
-                    results[i]['summary'].append('%s\t%s\t' % (seq_name,
-                                                               subject_id_a))
+                    results[i]['summary_fh'].write('%s\t%s\t\n' % (
+                        seq_name, subject_id_a))
                 db_seqs_counts_a[subject_id_a] += 1
             else:
                 results[i]['db_other'] += 1
-                results[i]['summary'].append('%s\n\t%s' % (seq_name, ''))
+                results[i]['summary_fh'].write('%s\n\t%s\n' % (seq_name, ''))
```

Comment:
Here there is no result for either the first or second. I guess it was a copy/paste, will change.